src/main/java/com/gooddata/visualizationObject/VisualizationObject.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright (C) 2007-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+
+
+package com.gooddata.visualizationObject;
+
+import com.fasterxml.jackson.annotation.*;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.As.WRAPPER_OBJECT;
+import static com.fasterxml.jackson.annotation.JsonTypeInfo.Id.NAME;
+import static com.gooddata.visualizationObject.CollectionType.*;
+import static java.util.stream.Collectors.toList;
+
+import com.gooddata.executeafm.UriObjQualifier;
+import com.gooddata.executeafm.afm.*;
+import com.gooddata.md.AbstractObj;
+import com.gooddata.md.Meta;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonTypeName(VisualizationObject.NAME)
+@JsonTypeInfo(include = WRAPPER_OBJECT, use = NAME)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class VisualizationObject extends AbstractObj {
+    static final String NAME = "visualizationObject";
+    private Content content;
+
+    @JsonCreator
+    public VisualizationObject(@JsonProperty("content") final Content content, @JsonProperty("meta") final Meta meta) {
+        super(meta);
+        this.content = content;
+    }
+
+    public Content getContent() {
+        return content;
+    }
+
+    @JsonIgnore
+    public List<Bucket> getBuckets() {
+        return getContent().getBuckets();
+    }
+
+    @JsonIgnore
+    public List<Measure> getMeasures() {
+        return getContent().getMeasures();
+    }
+
+    @JsonIgnore
+    public List<Measure> getSimpleMeasures() {
+        return getMeasures().stream()
+                .filter(measure -> measure.getDefinition() instanceof SimpleMeasureDefinition)
+                .collect(toList());
+    }
+
+    @JsonIgnore
+    public List<VisualizationAttribute> getAttributes() {
+        return getContent().getAttributes();
+    }
+
+    @JsonIgnore
+    public VisualizationAttribute getAttributeFromCollection(final CollectionType type) {
+        Bucket collectionBucket = getContent().getBuckets().stream()
+                .filter(bucket -> bucket.getLocalIdentifier().equals(type.toString()))
+                .findFirst()
+                .orElse(null);
+        if (collectionBucket == null) {
```

Comment:
please clean up a bit, checking for null, possible returned a line above from Optional is not very readable.